zeus/tasks/cleanup_artifacts.py

```diff
@@ -4,15 +4,15 @@
 
 from zeus.config import celery, db
 from zeus.constants import Status
-from zeus.models import Artifact
+from zeus.models import Artifact, Job
 from zeus.utils import timezone
 from zeus.utils.sentry import Span
 
 from .process_artifact import process_artifact
 
 
 @celery.task(name="zeus.cleanup_artifacts", time_limit=300)
-def cleanup_artifacts(task_limit=100):
+def cleanup_artifacts(task_limit=100, _process_artifact=process_artifact):
```

Comment:
was easier than patching due to duplicate module+func names and not being able to target them w/ mocker.patch